styles.css

```diff
@@ -48,4 +48,28 @@ audio {
 div#legend {
   text-align: center;
   vertical-align: middle;
-}
\ No newline at end of file
+}
+
+#chroma:before {
+  content: 'Chroma bands:';
```

Comment:
this is a real nitpick but it would be slightly better for accessibility to put this text in the HTML, or add add it in an `aria-label` attribute on the div, but absolutely not a deal breaker :P